resources/views/Widgets/Customer/AddressWidget.json

```diff
@@ -413,6 +413,13 @@
                     }
                 ]
             }
+        },
+        "spacing":{
```

Comment:
Du kannst die Definition des Containers mit in die globale json-Datei übernehmen und dann pro Widget nur noch folgende Angabe einfügen:
```
"spacing": "@include(spacing)"
```